bin/setup

```diff
@@ -39,7 +39,4 @@ FileUtils.chdir APP_ROOT do
 
   puts "\n== Removing old logs and tempfiles =="
   system! "bin/rails log:clear tmp:clear"
-
-  puts "\n== Restarting application server =="
-  system! "bin/rails restart"
```

Comment:
This is slowing down docker-base setups. I also personally don't need this, and it usually adds a bit of unnecessary time whenever I run `bin/setup`. Please let me know if this is a bad idea.